service/src/main/kotlin/fi/espoo/evaka/application/ApplicationQueries.kt

```diff
@@ -449,7 +449,7 @@ fun fetchApplicationSummariesForChild(h: Handle, childId: UUID): List<PersonAppl
     val sql =
         """
         SELECT
-            a.id, a.preferredUnit, a.startDate, a.sentDate, a.document->>'type' AS type,
+            a.id, a.preferredUnit, a.preferredStartDate AS startDate, a.sentDate, a.document->>'type' AS type,
```

Comment:
Should this field name be refactored for clarity?

Reply:
did some renaming, is that how you meant?

Reply:
Looks good to me!